HealthMonitoring.AcceptanceTests/Scenarios/Endpoint_registration.cs

```diff
@@ -76,13 +77,13 @@ public void Deleting_inexistent_endpoint()
         }
 
         [Scenario]
-        public void UpdatingEndpointTags()
+        public void Updating_endpoint_tags()
         {
             Runner.RunScenario(
                 _ => Given_a_monitor_api_client(),
                 _ => Given_endpoint_with_name_address_group_and_monitor_is_registered("my_name", "http://localhost:3333/status", "my_group", MonitorTypes.HttpJson),
                 _ => When_client_requests_tags_updating_via_url($"api/endpoints/{_identifier}/tags", new [] {"tag1", "tag2"}),
-                _ => Then_endpoint_tags_should_be_updated(new[] { "tag1", "tag2" })
+                _ => Then_status_should_be_returned(HttpStatusCode.OK)
```

Comment:
I am going to merge the change, but there should be more steps like:
_ => When_client_requests_endpoint_details(),
_ => Then_the_endpoint_tags_should_be_TAGS("tag1","tag2")

void Then_the_endpoint_tags_should_be_TAGS(params string[]tags)